test/tests/functional/pbs_multi_sched.py

```diff
@@ -122,6 +122,78 @@ def check_vnodes(self, j, vnodes, jid):
                 self.assertFalse(True, str(vnode) +
                                  " is not in exec_vnode list as expected")
 
+    def test_job_sort_formula_multisched(self):
+        """
+        Test that job_sort_formula can be set for each sched
+        """
+        self.common_setup()
+
+        self.server.expect(SCHED, 'job_sort_formula', id='sc1', op=UNSET)
+        self.server.expect(SCHED, 'job_sort_formula', id='sc2', op=UNSET)
+        self.server.expect(SCHED, 'job_sort_formula', id='sc3', op=UNSET)
+        self.server.expect(SERVER, 'job_sort_formula', op=UNSET)
```

Comment:
No reason to do this.  The system is reverted back to the default state in setUp().